docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml

```
@@ -0,0 +1,92 @@
+      - question: Can my Outlook add-in get an exception to finish the migration after Oct 2024?
+        answer: |
+          Unfortunately, no. We recognize that this migration is happening quickly, but we cannot offer exceptions to any partners. Migrating away from legacy tokens by October 2024 is a commitment to customers and their security teams. This effort will greatly improveimprove protection of customers' vital email data.
+      - question: Does my Outlook add-in have to migrate to NAA?
+        answer: |
+          No, Outlook add-ins don't have to use NAA. As long as add-ins are not using legacy Exchange tokens, they will not be affected by the deprecation of the legacy tokens.If add-ins are not using legacy Exchange tokens, they won’t be affected by their deprecation. Custom work using MSAL.js or other SSO methods that rely on Entra ID will still work after October 2024, as long as your add-in no longer relies on legacy Exchange user identity tokens and callback tokens.
+      - question: How do I know if my Outlook add-in relies on legacy tokens?
+        answer: |
+          To find out whether your add-in uses legacy Exchange user identity tokens and callback tokens, you can search your code for calls to the following APIs.
+
+          - `makeEwsRequestAsync`
+          - `getUserIdentityTokenAsync`
+          - `getCallbackTokenAsync`
+
+          If your add-in calls any of these APIs, you should adopt NAA and migrate to using Entra ID tokens to access Microsoft Graph instead.
+      - question: Which Outlook add-ins are in scope?
+        answer: |
+          Many of our most important add-ins are in scope. If your add-in is using EWS or Outlook REST to access Exchange Online resources, it almost certainly needs to migrate off of legacy Outlook tokens to NAA.
+          If your add-in is for Exchange on-premises only (for example, Exchange 2019), it is not affected by this announcement.
+      - question: What will happen to my Outlook add-ins if I don't migrate to NAA?
+        answer: |
+          If you don't migrate your Outlook add-ins to NAA, they will stop working as expected in Exchange Online. Beginning in April 2024, Exchange Online admins will have the abilitybe able to block legacy token issuance in their tenants. In October 2024, Exchange Online will block legacy token issuance by default in all tenants. Any add-in that uses legacy tokens will not be able to access Exchange online resources.
```

```suggestion
          If you don't migrate your Outlook add-ins to NAA, they will stop working as expected in Exchange Online. In October 2024, Exchange Online will block legacy token issuance by default in all tenants. Any add-in that uses legacy tokens will not be able to access Exchange online resources.
```
Removing the mention of admin controls since it's not available yet -- we should add it back in in the coming weeks